test/such/f_metadata.clj

```diff
@@ -40,8 +39,7 @@
 
   (fact "no arguments are required"
     (let [oo (meta/assoc o)]
-      (meta oo) => (meta o)
-      (identical? oo o) => false)))
```

Comment:
due to updating to clojure 1.10.0 (https://clojure.atlassian.net/browse/CLJ-2362)
I'm just going to remove the test given that I can't think of why it is particularly important